src/helpers/axiosWithAuth.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+export default function axiosWithAuth() {
+  const token = localStorage.getItem("token");
+  const axiosInstance = axios.create({
+    baseURL: "http://flashdecks-staging.herokuapp.com/",
+    headers: {
+      Authorization: token,
+    },
+  });
+  return axiosInstance;
+}
```

Comment:
Isn't this handled in src/utils/axios.js already?

Reply:
Not my code